### ISSUE TITLE
Add term prominent lesser trochanter

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10464]
**Preferred term label:**
Prominent lesser trochanter

**Synonyms**
Large trochanter minor

**Definition (free text, please give PubMed ID)**
Large lesser trochanter.

Comment: prominent lesser trochanters in combination with short femoral necks and medial metaphyseal beaking forms a “monkeywrench” or “Swedish- key” appearance, which is characteristic for Desbuguois dysplasia

Smithson S. (2019). Jürgen W. Spranger, Paula W. Brill, Christine Hall, Gen Nishimura, Andrea Superti-Furga, and Sheila Unger: Bone dysplasias: an atlas of genetic disorders of skeletal development: Fourth Edition, ISBN: 9780190626655. Human Genetics, 138(7), 787. https://doi.org/10.1007/s00439-019-02038-0

**Parent term (use hpo.jax.org/app)**
Abnormal femoral neck/head morphology HP:0003366

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Desbuquois dysplasia 2 (omim615777)
For example, PMID27030147

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000816
- label: Prominent lesser trochanter
- parents: [Abnormal femoral neck/head morphology (HP:0003366)](https://hpo.jax.org/app/browse/term/HP:0003366)
- pmid: PMID:27030147
- def: Abnormally large lesser trochanter.
- comment: Prominent lesser trochanters in combination with short femoral necks and medial metaphyseal beaking forms a monkeywrench or Swedish- key appearance, which is characteristic for Desbuguois dysplasia .
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10464
- synonyms:
